pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -45,8 +54,18 @@ func New(store storage.Store, prov provider) *Service {
 
 // Handle didexchange msg
 func (s *Service) Handle(msg dispatcher.DIDCommMsg) error {
-	// TODO add Handle logic
-	return nil
+	switch msg.Type {
```

Comment:
please leave `Handle` untouched for now. PR #183 is laying the groundwork for this method

Reply:
done